Makefile

```diff
@@ -252,6 +252,23 @@ test-e2e: KUSTOMIZE_BUILD_DIR := config/overlays/e2e
 test-e2e: GO_BUILD_EXTRA_FLAGS := -cover
 test-e2e: run image-registry e2e e2e-coverage kind-clean #HELP Run e2e test suite on local kind cluster
 
+.PHONY: test-e2e-metrics
+test-e2e-metrics: KIND_CLUSTER_NAME := operator-controller-e2e
+test-e2e-metrics: KUSTOMIZE_BUILD_DIR := config/overlays/e2e
+test-e2e-metrics: GO_BUILD_EXTRA_FLAGS := -cover
+test-e2e-metrics: run image-registry prometheus e2e e2e-coverage kind-clean #HELP Run e2e test suite on local kind cluster with prometheus installed
+
+.PHONY: prometheus
+prometheus: PROMETHEUS_NAMESPACE := prometheus
+prometheus: TMPDIR := $(shell mktemp -d)
```

Comment:
Do you delete this `$(TMPDIR)`?

Reply:
`mktemp` creates a directory in `/tmp` - if you think it's necessary to explicitly delete the directory in there we can do that easily :)

Reply:
yeah, it's bad to not delete, if you want to make sure it goes away in the same target, you could add:
`trap 'echo "Cleaning up $$tmpdir for $@"; rm -rf "$$tmpdir"' EXIT;`

Reply:
Done! Please feel free to un-resolve if the solution needs work.